lib/kb_SPAdes/kb_SPAdesClient.py

```diff
@@ -23,7 +23,7 @@ def __init__(
             self, url=None, timeout=30 * 60, user_id=None,
             password=None, token=None, ignore_authrc=False,
             trust_all_ssl_certificates=False,
-            auth_svc='https://kbase.us/services/authorization/Sessions/Login'):
+            auth_svc='https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login'):
```

Comment:
Are you sure you want to use the ci environment auth service here? The path doesn't look like its test code.

Reply:
ditto this one.  why did make add CI ?!?!?!?!

Reply:
manually fixed them and then ran make (on docker02).  it put CI back and legacy back.  I manually fixed it and won't run make again!

Reply:
This is also from recompiling the client. It doesn't really matter - this url is only used for translating a user/pwd combo to a token, which no longer happens. Really the clients should be updated to remove it.

Reply:
@MrCreosote so this one should be https://kbase.us/services/auth/api/legacy/KBase/Sessions/Login ?

Reply:
It should be whatever `make` generates, but it doesn't really matter since it should never be used.

Reply:
Again it'll get blown away on registration anyway.